app/screens/channel/channel.test.tsx

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
This test file feels a little off.  The two tests are duplicated?

And there are no story to tell here.

If a test is saying that there's no post when switching teams, can you show there are post when not switching team?

I guess there needs to be some sort of a base.  Like what do I expect the channel file to display when it has basic props.

Then go from there...

Reply:
I'm deleting this test for now, its not core to the changes here.